src/utils/csv.ts

```diff
@@ -0,0 +1,57 @@
+import { logDebug } from 'utils/miscellaneous'
+
+// TODO: can I make sure - using types - that headers length matches transformer return type length?
```

Comment:
Can't typecheck length unless we do tuples, and we can't do tuples
A compromise could be adding `assert` for a runtime check inside `toCsv`

Reply:
Wrote down my idea to address that here https://github.com/gnosis/dex-react/pull/1012#discussion_r432756355